packages/configure-splash-screen/src/__tests__/cli-command.test.ts

```diff
@@ -0,0 +1,138 @@
+import configureAndroid from '../android';
```

Comment:
We should rename all of these tests to `-test` after this PR is merged. https://github.com/expo/expo-cli/issues/2591